<launch>
  <arg name="config_path" default="$(find config)/vehicle" />
  <arg name="sensor_meta" default="$(arg config_path)/sensor/calibrated_sensor.pb.txt" />
  <arg name= "rslidar_path" default="$(find drivers_robosense)" />
  <group ns="/sensor/lidar/robosense">
    <node pkg="drivers_robosense" name="drivers_robosense_node" 
      type="drivers_robosense_node" output="screen"
      args="--logtostderr=0 --stderrthreshold=1 --v=5
          --sensor_meta_path=$(arg sensor_meta)">
	  <param name="rslidar_config_path" type="string" value="$(arg rslidar_path)" />
    </node>
    <!-- rviz -->
    <node pkg="rviz" name="rviz" type="rviz" args="-d $(find drivers_robosense)/rviz/rviz.rviz" />
  </group>
</launch>
